src/transformers/tokenization_utils_base.py

```diff
@@ -1932,6 +1932,7 @@ def _from_pretrained(
                     cache_dir=cache_dir,
                     local_files_only=local_files_only,
                     _commit_hash=_commit_hash,
+                    trust_remote_code=kwargs.get('trust_remote_code',False)
```

Comment:
```suggestion
                    **(copy.deepcopy(kwargs)),
```
think this should prevent other issues no? 
The `trust_remote_code` kwarg is popped in the AutoConfig as well

Reply:
Indeed, if both of the from_pretrained interfaces are well-defined and consistently used, it would be better to transfer all the keyword arguments!